core/src/main/scala/org/http4s/parser/Rfc3986Parser.scala

```diff
@@ -69,9 +70,7 @@ private[http4s] trait Rfc3986Parser
   def Host: Rule1[org.http4s.Uri.Host] = rule {
     // format: off
     ipv4Address |
-    (IpLiteral | capture(IpV6Address)) ~> { s: String =>
-      org.http4s.Uri.IPv6(s.ci)
-    } |
+    "[" ~ ipv6Address ~ "]" |
```

Comment:
I think we temporarily lost support for IPvFuture here.  There wasn't a test, and I highly doubt anyone cares, but it needs to be brought back for compliance sake.